recipe/bld.bat

```diff
@@ -1,9 +1,34 @@
 @echo on
-setlocal enabledelayedexpansion
-
-:: copy %SRC_DIR%/base somewhere to be able to hard-reset state in build-output.bat
-mkdir .\backup
-robocopy base backup /E >nul
-:: for whatever reason, robocopy returns 1 in case of success, see
-:: https://learn.microsoft.com/en-us/troubleshoot/windows-server/backup-and-storage/return-codes-used-robocopy-utility
-if %ERRORLEVEL% neq 1 exit 1
+
+mkdir builddir
+
+:: check if clang-cl is on path as required
+clang-cl.exe --version
+if %ERRORLEVEL% neq 0 exit 1
+
+:: set compilers to clang-cl
+set "CC=clang-cl"
+set "CXX=clang-cl"
+
+:: flang 17 still uses "temporary" name
+set "FC=flang-new"
+
+:: set up clang-cl correctly, see
+:: https://github.com/conda-forge/clang-win-activation-feedstock/blob/main/recipe/activate-clang_win-64.sh
+set "CPPFLAGS=-D_CRT_SECURE_NO_WARNINGS -D_MT -D_DLL --target=x86_64-pc-windows-msvc -Xclang --dependent-lib=msvcrt -fuse-ld=lld"
+set "FFLAGS=-D_CRT_SECURE_NO_WARNINGS -D_MT -D_DLL --target=x86_64-pc-windows-msvc -nostdlib"
```

Comment:
You probably need `--dependent-lib=msvcrt` here as well since libraries to link to are hardcoded in compilation objects on windows.

Reply:
When I tried it with `-Xclang` it gets ignored, when I tried it with `-Xflang` it fails the build, see also https://github.com/llvm/llvm-project/issues/63741

Reply:
For reference, here's a [CI run](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=791496&view=logs&j=171a126d-c574-5c8c-1269-ff3b989e923d&t=623c2ba8-c4f1-582d-6c78-1c7699e2e0a2) where I had tried this in #246